DFC.ServiceTaxonomy.ContentApproval/Permissions/CanPerformReviewPermissions.cs

```diff
@@ -10,6 +10,7 @@ public class CanPerformReviewPermissions : IPermissionProvider
         public static readonly Permission CanPerformContentDesignReviewPermission = new Permission(nameof(CanPerformContentDesignReviewPermission), "Can perform content design review");
         public static readonly Permission CanPerformSMEReviewPermission = new Permission(nameof(CanPerformSMEReviewPermission), "Can perform SME review");
         public static readonly Permission CanPerformUXReviewPermission = new Permission(nameof(CanPerformUXReviewPermission), "Can perform UX review");
+        public static readonly Permission CanPerformReviewPermission = new Permission(nameof(CanPerformUXReviewPermission), "Can perform UX review", new []{CanPerformUXReviewPermission, CanPerformSMEReviewPermission, CanPerformContentDesignReviewPermission, CanPerformApprovalPermissions.CanPerformStakeholderApprovalPermission});
```

Comment:
Should the string be "Can perform review"?

Reply:
Agree, copy/paste fail